docs/management/managing-remote-clusters.asciidoc

```diff
@@ -1,26 +1,64 @@
-[[managing-remote-clusters]]
-== Managing Remote Clusters
+[[working-remote-clusters]]
+== Working with remote clusters
+
+{kib} *Management* provides user interfaces for working with data from remote 
+clusters and managing the cross cluster replication process. Replicating indices from a 
+leader remote cluster to a follower index in a local cluster can be used to provide 
+remote backups for disaster recovery or for geo-proximite copies of data.
+
+Before using these features, you should be familiar with the following concepts:
 
-{kib} *Management* provides two user interfaces for working with data from remote 
-clusters. 
+* {xpack-ref}/xpack-ccr.html[Cross cluster replication]
+* {ref}/modules-cross-cluster-search.html[Cross cluster search]
+* {xpack-ref}/cross-cluster-configuring.html[Cross cluster security requirements]
+
+[float]
+[[managing-remote-clusters]]
+== Managing remote clusters 
 
 *Remote Clusters* helps you manage remote clusters for use with 
-{ref}/modules-cross-cluster-search.html[cross cluster search] and 
-{xpack-ref}/xpack-ccr.html[cross cluster replication]. You can add and remove remote 
+cross cluster search and cross cluster replication. You can add and remove remote 
 clusters and check their connectivity. 
 
+Before you use this feature, you should be familiar with the concept of 
+{ref}/modules-remote-clusters.html[remote clusters].  
+ 
 Go to *Management > Elasticsearch > Remote Clusters* to get started. 
 
 [role="screenshot"]
 image::images/add_remote_cluster.png[][UI for adding a remote cluster]
 
+Once a remote cluster is registered, you can use the tools under *Cross Cluster Replication* 
```

Comment:
Possibly not, I was trying to show the connection between the UIs, and in particular, that setting up 
 a remote cluster is a prereq to using any features under the CCR menu. But it might be redundant.